### ISSUE TITLE
Why is gradient accumulation disabled when training from scratch?

### ISSUE THREAD

**Issue by IrvingF7** [openvla/openvla#198]
I noticed that in the vla-training strategy, gradient accumulation is prohibited. 

Is there any reason or catch for this? I also see that `run_training` has logic for gradient accumulation implemented, but `run_vla_training` doesn't have corresponding logic.

I really want to match the default batch size setup provided, but I am currently constrained by VRAM, so I am looking to do some small gradient accumulation.

I may end up modifying `run_vla_training` but I don't want to commit to it if it turns out there are some catches that made you guys disable gradient accumulation in the first place.

Anyway, thank you guys for open-sourcing this, and I am very grateful for this codebase.

**Comment by choucaicai** [comment]
same question
